doc/architecture/decisions/0001-record-architecture-decisions.md

```diff
@@ -0,0 +1,19 @@
+# 1. Record architecture decisions
```

Comment:
Do we need this or should it live in the top level README of how to raise changes for this repo. 

One this that is difficult with https://github.com/ONSdigital/software-engineer-community is that there is no agreed process documented for approving these things. It might be worth describing how we would do this with the process i.e. Tech leads meeting every fortnight or something like that?